### PR TITLE
some hints for beginners

### PR DESCRIPTION
**Pull request by marchon**
I added:
- some changes to the README for newbies
  - note to run npm install 
  - how to install and run the linter
- a linting example in index.js
- better defaults to .gitignore from [https://github.com/github/gitignore/blob/master/Node.gitignore]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,13 +50,17 @@
...
+    "react-redux": "^4.0.0",
     "react-transform-catch-errors": "^1.0.0",
     "react-transform-hmr": "^1.0.1",
     "redbox-react": "^1.2.0",
+    "redux": "^3.0.2",
     "style-loader": "^0.13.0",
     "url-loader": "^0.5.7",
     "webpack": "^1.12.9",
     "webpack-dev-middleware": "^1.4.0",
+    "webpack-dev-server": "^1.14.0",
```
Comment (chicoxyzzy) [inline package.json:63]: this boilerplate uses `webpack-dev-middleware`. so `webpack-dev-server` is unnecessary here

File: package.json [hunk 2]
```diff
@@ -52,13 +50,17 @@
     "html-webpack-plugin": "^1.7.0",
     "json-loader": "^0.5.4",
     "postcss-loader": "^0.8.0",
+    "react-hot-loader": "^1.3.0",
```
Comment (chicoxyzzy) [inline package.json:53]: we use `react-transform` instead of `react-hot-loader`

File: package.json [hunk 3]
```diff
@@ -43,6 +39,8 @@
     "babel-loader": "^5.4.0",
     "babel-plugin-react-transform": "^1.0.3",
     "css-loader": "^0.23.0",
+    "debug": "^2.2.0",
+    "debug-dude": "^1.0.1",
```
Comment (chicoxyzzy) [inline package.json:43]: this two, `react-redux` and `redux` should be in dependencies

**Comment by chicoxyzzy** [comment]
Thank you! Can you please rebase from master and fix some issues in `package.json`?

**Comment by marchon** [comment]
I would be delighted.

G.

On Mon, Nov 30, 2015 at 2:06 PM, Sergey Rubanov notifications@github.com
wrote:

> Thank you! Can you please rebase from master and fix some issues in
> package.json?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/chicoxyzzy/react-for-hipsters-boilerplate/pull/82#issuecomment-160726801
> .

## 

P THINK BEFORE PRINTING: is it really necessary?

This e-mail and its attachments are confidential and solely for the
intended addressee(s). Do not share or use them without approval. If
received in error, contact the sender
and delete them.

**Pull request closed without merging**